.gitignore

```diff
@@ -7,6 +7,7 @@ dist
 globus_sdk.egg-info
 eggs/
 .eggs/
+.idea
```

Comment:
part of me wants to say this should be its own commit or something, but its pretty innocuous so this is probably fine.